torchci/lib/bot/utils.ts

```diff
@@ -79,3 +80,11 @@ export class CachedIssueTracker extends CachedConfigTracker {
     return this.repoIssues[key];
   }
 }
+
+export async function fetchJSON(path: string): Promise<any> {
+  const result = await urllib.request(path);
+  if (result.res.statusCode !== 200) {
+    return;
```

Comment:
nit:  Could you add a comment explaining why it's safe to ignore failed response codes?  (I do agree that it's safe to do so)

Reply:
added st about what the returned value is if it fails